CODEOWNERS

```
@@ -422,7 +422,7 @@
 /dts/arm/nxp/                             @mmahadevan108 @dleach02
 /dts/arm/nxp/nxp_s32*                     @manuargue
 /dts/arm/microchip/                       @franciscomunoz @albertofloyd @sjvasanth1
-/dts/arm/rpi_pico/                        @yonsch
+/dts/arm/raspberry/rpi_pico/              @yonsch
```

Wrong vendor prefix:
```suggestion
/dts/arm/raspberrypi/rpi_pico/            @yonsch
```

---

> I would just drop this change altogether. See #82230

Sure, if #82230 gets merged first (which I guess it will).

But until then, if this PR contains a change to `CODEOWNERS` then such change should be correct.

---

My point was that the CODEOWNERS file is plain stale anyway and full of incorrect entries (it wasn't even covered with hwmv2 migration and nobody noticed/bothered... https://github.com/zephyrproject-rtos/zephyr/commit/8dc3f856229ce083c956aa301c31a23e65bd8cd8#diff-fcf14c4b7b34fe7a11916195871ae66a59be87a395f28db73e345ebdc828085b). Dropping this change is effectively a simple way to avoid a possible future merge conflict IMO. 

BTW if the change stays (which I really recommend it doesn't), why is the code owner not being updated to @soburi?

---

> Dropping this change is effectively a simple way to avoid a possible future merge conflict IMO.

Agreed. We should just wait the ~48 hours for #82230 . Absolute worse case scenario would be a known-to-be-wrong file is made a tiny bit more wrong, before being deleted.

> if the change stays (which I really recommend it doesn't), why is the code owner not being updated to @soburi?

My assumption was the file was correct at the point I changed it, which it turns out was not the case, as you've pointed out.

---

This was a blocking request for change, but now it's overtaken by events as the underlying file has been deleted.

I hope this conversation can now be resolved, and we get some approvals against this.